browse/templates/cookies.html

```diff
@@ -119,19 +118,24 @@ <h2>Opt-in Data Collection for Research (Cornell University)</h2>
   checkOptInParticipation();
 
   // Generate UUID
-  document.getElementById('opt-in-registration').addEventListener('click', event => {
-  event.preventDefault();
-  const uuid = crypto.randomUUID(); // Generate a unique ID
-  document.cookie = `opt-in-tracking=${uuid}; Path=/; Max-Age=31536000`; // 1 year
-  checkOptInParticipation(); // Update UI
-  });
-
-
-  document.getElementById('opt-out').addEventListener('click', event => {
+  const regBtn = document.getElementById('opt-in-registration');
+  if (regBtn) {
+    document.getElementById('opt-in-registration').addEventListener('click', event => {
     event.preventDefault();
-    document.cookie = 'opt-in-tracking=; Path=/; Expires=Thu, 01 Jan 1970 00:00:01 GMT;';
-    checkOptInParticipation();
-  });
+    const uuid = crypto.randomUUID(); // Generate a unique ID
+    document.cookie = `opt-in-tracking=${uuid}; Path=/; Max-Age=31536000`; // 1 year
+    checkOptInParticipation(); // Update UI
+    });
+  }
+
+  const optOutBtn = document.getElementById('opt-out');
+  if (optOutBtn) {
+      document.getElementById('opt-out').addEventListener('click', event => {
```

Comment:
```suggestion
      optOutBtn.addEventListener('click', event => {
```

Reply:
I eliminated extra getElementById in two spots.